frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkAttachmentDefinitionsForm.tsx

```diff
@@ -42,18 +42,12 @@ const buildConfig = (name, networkType, typeParamsData): NetworkAttachmentDefini
   }
 
   if (networkType === 'cnv-bridge') {
-    // cnv-bridge should not define type on root
-    delete config.type;
-
-    config.plugins = [
-      {
-        type: 'cnv-bridge',
-        bridge: _.get(typeParamsData, 'bridge.value', ''),
-        vlan: parseInt(typeParamsData?.vlanTagNum?.value, 10) || undefined,
-        ipam,
-      },
-      { type: 'cnv-tuning' },
-    ];
+    Object.assign(config, {
```

Comment:
Have you confirmed that the `cnv-bridge` property isn't being set at the root of the config object?

Reply:
Hi, according to @phoracek , we dont need cnv-tuning, and when removing it we don't need plugins and cnv-bridge can set in config directly.

Reply:
The cnv-bridge `type` is still needed, it just sits on the root.

Reply:
Ah, ok. Thanks for the clarification!

Reply:
please don't use `Object.assign` here, use simple assing
for example
``` ts
config.bridge = typeParamsData?.bridge?.value || '';
config.vlan = parseInt(typeParamsData?.vlanTagNum?.value, 10) || undefined;
config.macspoofchk = typeParamsData?.macspoofchk?.value ||  false;
```

Reply:
changed to direct assinging